File: src/main/java/com/qsoftware/bubbles/common/cursor/BlankCursorRenderer.java
package com.qsoftware.bubbles.common.cursor;

import com.qsoftware.bubbles.common.renderer.CursorRenderer;

import java.awt.*;

public class BlankCursorRenderer extends CursorRenderer {
    public BlankCursorRenderer() {
        super("blank_cursor");
    }

    @Override
    public void draw(Graphics g) {

    }
}

File: src/main/kotlin/qtech/bubbles/common/cursor/BlankCursorRenderer.kt
package qtech.bubbles.common.cursor

import qtech.bubbles.common.renderer.CursorRenderer
import qtech.hydro.GraphicsProcessor
import java.awt.Point

class BlankCursorRenderer : CursorRenderer("blank_cursor") {
    override val hotSpot: Point
        get() = Point(16, 16)

    override fun draw(g: GraphicsProcessor) {}
}